### ISSUE TITLE
Not compatible with Python 3.11.1

### ISSUE THREAD

**Issue by abrignoni** [ydkhatri/pyliblzfse#3]
<img width="566" alt="Screenshot 2023-01-10 at 6 49 37 PM" src="https://user-images.githubusercontent.com/28718987/211686043-646c5622-70e3-4eab-a863-7b928931c6b0.png">

**Comment by ydkhatri** [comment]
Hmm, I'll look into it later today.

**Comment by abrignoni** [comment]
Incompatible on macOS.

**Comment by ydkhatri** [comment]
No issues on python 3.11.1 for me, on macOS 12.6. Are you running macOS 13 (Ventura) ?

**Comment by abrignoni** [comment]
Ventura 13.1

**Comment by ydkhatri** [comment]
I'm still not able to reproduce the error on Ventura 13.1 and Python 3.11.1. 
But I am on an intel mac. Are you on ARM platform (M1/M2)?

**Comment by abrignoni** [comment]
I'm on Intel. 🤔 Maybe I'm wrong? This whole issue is that we tried to run
iLEAAP on 3.11.1 and gave us errors that lead us to the pylonlzfse library.
Or at least I thought so...

**Comment by ydkhatri** [comment]
Ok, so this is weird. Not sure what the problem is, but I can at least see it now. On a VM, I have no problems. But on host , same error as you get. I'll try to debug it now.

**Comment by ydkhatri** [comment]
The only difference I can remember is that on the host I updated xcode libraries. Did not update xcode on the VM.

**Comment by abrignoni** [comment]
Weird. I don't think I have xcode installed on my Mac btw. Hopefully the
issue can be identified. 🤞

**Comment by ydkhatri** [comment]
Ok, so updating Xcode via `xcode-select --install` on the system that had the issue seems to have fixed it. This is required to compile the code. (Nothing's changed, its always been this way).

I'd try the above (xcode) and then installing in a virtual environment. That should narrow down if its an environment issue or something else.

```
(venv) ykhatri@macsystem temp % pip install pyliblzfse
Collecting pyliblzfse
  Using cached pyliblzfse-0.4.1.tar.gz (47 kB)
  Preparing metadata (setup.py) ... done
Installing collected packages: pyliblzfse
  DEPRECATION: pyliblzfse is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for pyliblzfse ... done
Successfully installed pyliblzfse-0.4.1

**Comment by abrignoni** [comment]
So I would need to install xcode on a windows environment for it to work?

**Comment by ydkhatri** [comment]
No, that is for macOS only. Are you having issues on Windows too?

**Comment by abrignoni** [comment]
Yes.

**Comment by ydkhatri** [comment]
I got no issues installing it on Windows 10 with Python 3.11.1.
```
(venv) c:\Python311\test>pip install pyliblzfse
Collecting pyliblzfse
  Downloading pyliblzfse-0.4.1.tar.gz (47 kB)
     ---------------------------------------- 47.7/47.7 kB 2.3 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Installing collected packages: pyliblzfse
  DEPRECATION: pyliblzfse is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for pyliblzfse ... done
Successfully installed pyliblzfse-0.4.1

[notice] A new release of pip available: 22.3.1 -> 23.0
[notice] To update, run: python.exe -m pip install --upgrade pip
```

**Comment by abrignoni** [comment]
Confirmed. Thank you for the macos fix. 🙏
